README.md

```diff
@@ -228,9 +228,17 @@ mv modules/* modules_bak/
 ```
 
 2. Start up the server
+Usually, you run:
 ```
 mvn openmrs-sdk:run -DserverId=[serverId]
 ```
+But due to a current bug in version 4.2.0 of the SDK, you should tell Maven to expliclity use
+the 4.1.1 release of the SDK when doing this initial run (you can you the standard openmrs-sdk:run
+after that)
+```
+mvn org.openmrs.maven.plugins:openmrs-sdk-maven-plugin:4.1.1:run -DserverId=[serverId]
+```
+
```

Comment:
This is fine assuming that run is the only goal implicated.  Personally, I just created a helper script that I use to ensure all of my sdk commands use the 4.1.1 release without having to remember or type out that massive thing.

I have a file called "sdk.sh":

```
#!/bin/bash

OP=$1
SERVER_ID=$2

mvn org.openmrs.maven.plugins:openmrs-sdk-maven-plugin:4.1.1:$OP -DserverId=$SERVER_ID ${@:3}
```
And then I have an alias setup in .bashrc:
`alias omrs=''/path/to/sdk.sh'`

And then I just run:
`omrs run mirebalais`
or
`omrs setup`
or whatever...